src/main/features/dashboard/views/macro/claimStatus/claimFullAdmission.njk

```diff
@@ -0,0 +1,116 @@
+{% from "externalLink.njk" import externalLink %}
+{% from "form.njk" import linkButton %}
+
+{% macro claimFullAdmissionForDefendantDashboard(claim) %}
+  {% if claim.response.paymentIntention.paymentDate %}
+    {% set paymentDate = claim.response.paymentIntention.paymentDate | date %}
+  {% endif %}
+
+  {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+    {% set paymentMethod = 'immediately' %}
+  {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
+    {% set paymentMethod = 'by ' + paymentDate %}
+  {% elseif  claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
+    {% set paymentMethod= 'in instalments' %}
+  {% endif %}
+  {{ t('You’ve admitted all of the claim and said you’ll pay the full amount {{ paymentMethod}}.', { paymentMethod: paymentMethod }) }}
+{% endmacro %}
+
+{% macro claimFullAdmissionForDefendantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('Your response to the claim') }}</h2>
+  {% set claimantName = claim.claimData.claimant.name %}
+  {% set amount = claim.totalAmountTillToday | numeral %}
+
+  {% if claim.response.paymentIntention.paymentDate %}
```

Comment:
The same as comment above, remove and move inside 'claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE' block.